### PR TITLE
[WIP] add offset to z direction of bbox

### PR DESCRIPTION
**Pull request by yuyu2172**

### CODE REVIEW

File: jsk_apc2016_common/include/sib/cloud_to_spatial_features.h [hunk 1]
```diff
@@ -18,6 +18,8 @@
 #include <pcl_ros/point_cloud.h>
 #include <pcl_ros/transforms.h>
 
+#define Z_OFFSET 0.01
```
Comment (pazeshun) [inline jsk_apc2016_common/include/sib/cloud_to_spatial_features.h]: Use `const` global variable instead of `#define`

**Comment by knorth55** [comment]
👍

File: jsk_apc2016_common/include/sib/cloud_to_spatial_features.h (hunk 1 above)
Comment (yuyu2172) [inline jsk_apc2016_common/include/sib/cloud_to_spatial_features.h]: fixed

**Comment by yuyu2172** [comment]
tuning params

**Comment by pazeshun** [comment]
+1 but we should know why this offset is needed.

**Comment by yuyu2172** [comment]
need_for_offeset

**Comment by yuyu2172** [comment]
After adjusting kinect and kiva_pod_base, this PR became unnecessary.

**Pull request closed without merging**